sh/tests/grades_test.sh

```diff
@@ -73,32 +58,11 @@ john
 49
 Eric
 65
-EOF
-    )
-    diff <(echo "$submitted") <(echo "$expected")
-
-    # Checking if it fails with invallid grades
-    submitted=$(
-        bash -c ""$script_dirS"/student/grades.sh 3" <<EOF
-Bob
+"
+challenge 3 "Bob
 90
 Alice
 150
 Eve
 55
-EOF
-    )
-    expected=$(
-        bash -c ""$script_dirS"/solutions/grades.sh 3" <<EOF
-Bob
-90
-Alice
-150
-Eve
-55
-EOF
-    )
-    diff <(echo "$submitted") <(echo "$expected")
-}
-
-challenge
+"
```

Comment:
- `0` as an imput for grade is not tested, it could be nice to add this case.
- also negative inputs are not tested.
- Input which is not a number for grade is also not tested

Reply:
done